### ISSUE TITLE
LC0071: "IsHandled or Boolean" not supported

### ISSUE THREAD

**Issue by tkroes** [StefanMaron/BusinessCentral.LinterCop#997]
Please correct me if I'm wrong, but I believe the following code should not trigger LC0071:
```al
IsHandled := IsHandled or Item.Blocked;
```
In this code there is no possibility of setting IsHandled to false from true, while setting it to true when it is false is possible. Would it be possible to add this scenario to the rule?

**Comment by Arthurvdv** [comment]
Thank you for creating this issue and indeed the rule raised a false positive on your example.

Credits to @ans-bar-bm for creating a pull-request to improve the rule on this (i'm merely the messenger here).

This should now be resolved in the Pre-release [v0.35.1](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.35.1) version of the LinterCop.

**Comment by Arthurvdv** [comment]
The version [v0.36.0](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.36.0) of LinterCop has been released as the latest, where this issue is now resolved.
